app/test_main.py

```diff
@@ -1,3 +1,36 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,expect_result",
+    [
+        (-15, -15, [0, 0]),
+        (0, 0, [0, 0]),
+        (1, 1, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (29, 29, [3, 3]),
+        (150, 150, [33, 27])
```

Comment:
Requested tests added except the one that covers different values for `cat_age` and `dog_age`. Encounter this test as well (e.g. passing 28 and 15, 14 and 29 to the `get_human_age` function to test that produced output is correct).